packages/protocol/contracts/L1/libs/LibTokenomics.sol

```diff
@@ -23,6 +23,21 @@ library LibTokenomics {
     // @dev Since we keep the base fee in 10*18 but the actual TKO token is in 10**8
     uint256 private constant SCALING_FROM_18_TO_TKO_DEC = 1e10;
 
+    /// Todo: Daniel and Brecht: This is an idea to replace current gasLimit and play around
+    /// with the idea of having it a (fine tuneable) parameter - for now (testing) as a constant.
+    /// @notice Decimal factor, meaning this is the variable which will be used to multiply
+    /// the basefee (a constant =  input.gasLimit).
+    ///
+    /// @dev To give more context, the fee and reward: (if the proof arrives at around target time)
+    /// IF DECIMAL_FACTOR = 1_000_000;
+    /// with proofTImeTarget 85 sec (current testnet) apprx.: 782.7 TKO is the reward (both with and without minting)
+    /// with proofTimeTarget is 20 mins (on mainnet): apprx.: 55.4 TKO (approx. both with and without minting)
+
+    /// IF DECIMAL_FACTOR = 100_000;
+    /// with proofTImeTarget 85 sec (current testnet) apprx.: 78.27 TKO is the reward
+    /// with proofTimeTarget is 20 mins (on mainnet): apprx.: 5.54 TKO
+    uint32 private constant DECIMAL_FACTOR = 100_000;
```

Comment:
To set reasonable starting values I think better to set some rough expected target values for the initial fee and calculate the corresponding `proofTimeIssued` value, that way there are fewer magic constants. From the python code:
```
basefee_proof = 10**8
proof_time_issued = (PROOF_TIME_TARGET * PROOF_ADJUSTMENT_QUOTIENT) * math.log(basefee_proof * (PROOF_TIME_TARGET * PROOF_ADJUSTMENT_QUOTIENT))
```

So if we're not interested into taking gas into account anymore (which I generally agree with because some of the costs are completely unrelated to L2 gas so not really a good estimate), we don't really _have_ to multiply with any factor, `basefee == fee` directly.